entity_history/models.py

```diff
@@ -21,7 +21,7 @@ class EntityRelationshipActivationEvent(models.Model):
     super_entity = models.ForeignKey(
         Entity, related_name='+', help_text='The super entity in the relationship that was activated / deactivated')
     time = models.DateTimeField(db_index=True, help_text='The time of the activation / deactivation')
-    was_activated = models.BooleanField(help_text='True if the entity was activated, false otherwise')
+    was_activated = models.BooleanField(default=False, help_text='True if the entity was activated, false otherwise')
```

Comment:
The default in Django 1.6 was `False`, and it was changed in 1.7 to `None`. A warning is generated if you don't enter anything, so I just set it to the past behavior. Would you prefer it be `None`?